framework/src/main/java/org/tron/core/services/filter/HttpInterceptor.java

```diff
@@ -66,28 +66,28 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
       if (EndpointCount.containsKey(endpoint)) {
         obj = EndpointCount.get(endpoint);
       } else {
-        obj.put(monitorMetric.TOTAL_REQUST, 0);
-        obj.put(monitorMetric.FAIL_REQUST, 0);
+        obj.put(metricsService.TOTAL_REQUST, 0);
```

Comment:
this is specific endpoint statics data, cannot apply metricsSever.TOTL_REQUEST since this is for total number of request, not a specific endpoint.